### DISCUSSION TITLE
About Managing Scope for Team Members

### DISCUSSION THREAD

**Discussion by Aomsub101** [supabase/supabase#30550] [Questions]
I have a question regarding access control for managing scopes assigned to invited team members within an organization. Is it possible to use PostgreSQL to manage these scopes? Or is there any other way than using an Enterprise plan? Thank you.

**Comment by silentworks** [comment]
The scopes you are referring to here are the access control for the dashboard itself, this isn't related to postgres at all. You can manage scopes on Team and Enterprise plan. https://supabase.com/docs/guides/platform/access-control#organization-scoped-roles-vs-project-scoped-roles
